tests/sdfg/race_conditions_check.py

```diff
@@ -0,0 +1,34 @@
+import dace
```

Comment:
1. missing copyright line, see [CONTRIBUTING.md](https://github.com/spcl/dace/blob/master/CONTRIBUTING.md)
2. filename is incorrect and won't be caught by pytest. As with other tests, please end filename with `_test.py`